java/README.md

```diff
@@ -0,0 +1,35 @@
+# Java Examples
+
+![Language-Support: Stable](https://img.shields.io/badge/language--support-stable-success.svg?style=for-the-badge)
+
+This section contains all the CDK code examples written in Java. To manage CDK packages in Java we use [Apache Maven](https://maven.apache.org/) (packages managed on [Central Repository](https://search.maven.org/)). For more information on using the CDK in Java, please see the [Developer Guide](https://docs.aws.amazon.com/cdk/latest/guide/work-with-cdk-java.html).
+
+## Running Examples
+
+To run a Java examples, execute the following:
+
+```
+$ npm install -g aws-cdk
+$ cd java/EXAMPLE_DIRECTORY
+$ mvn package
```

Comment:
You shouldn't need to explicitly build anymore.

Reply:
```suggestion
```